src/allmydata/webish.py

```diff
@@ -44,6 +44,42 @@
     StoragePlugins,
 )
 
+
+if PY2:
+    FileUploadFieldStorage = FieldStorage
+else:
+    class FileUploadFieldStorage(FieldStorage):
+        """
+        Do terrible things to ensure files are still bytes.
```

Comment:
[Looks like](https://app.circleci.com/pipelines/github/tahoe-lafs/tahoe-lafs/1864/workflows/2677d95a-d206-4a4b-af5c-6edfb10dcd71/jobs/44212) CI agrees with you that they're terrible:

```
typechecks create: /root/project/.tox/typechecks
typechecks installdeps: mypy, git+https://github.com/Shoobx/mypy-zope, git+https://github.com/warner/foolscap
typechecks installed: attrs==20.3.0,Automat==20.2.0,cffi==1.14.4,constantly==15.1.0,cryptography==3.3.1,foolscap @ git+https://github.com/warner/foolscap@291df027d5770f5edd6df01368de3e2cf273ccdf,hyperlink==21.0.0,idna==3.1,incremental==17.5.0,mypy==0.800,mypy-extensions==0.4.3,mypy-zope @ git+https://github.com/Shoobx/mypy-zope@ca0fe617f0303e60faf944e02b47a6b2717c3921,pyasn1==0.4.8,pyasn1-modules==0.2.8,pycparser==2.20,PyHamcrest==2.0.2,pyOpenSSL==20.0.1,service-identity==18.1.0,six==1.15.0,Twisted==20.3.0,typed-ast==1.4.2,typing-extensions==3.7.4.3,zope.event==4.5.0,zope.interface==5.2.0,zope.schema==6.0.1
typechecks run-test-pre: PYTHONHASHSEED='787671381'
typechecks run-test: commands[0] | mypy src
src/allmydata/webish.py:68: error: Signature of "filename" incompatible with supertype "FieldStorage"
Found 1 error in 1 file (checked 284 source files)
ERROR: InvocationError for command /root/project/.tox/typechecks/bin/mypy src (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   typechecks: commands failed
```

Reply:
I just told mypy to not type check that function :(